makefile

```diff
@@ -14,10 +14,20 @@ build: clear install
 	@rm -rf dist
 	@NODE_ENV=production webpack --bail --color --config __config__/webpack/webpack.config.js
 
+build-ci: install
```

Comment:
do we need a seperate task for this? `build` should be ok?

Reply:
Not sure. It sets the env variable so I guess it's testing the webpack bundling in production mode? Don't know how it works

Reply:
we don't need CI env variable to build on CI, that's why it was removed before